cni-plugin/pkg/install/install.go

```diff
@@ -502,12 +499,16 @@ current-context: calico-context`
 		data = strings.Replace(data, "__TLS_CFG__", ca, -1)
 	}
 
-	if err := os.WriteFile("/host/etc/cni/net.d/calico-kubeconfig", []byte(data), 0600); err != nil {
+	if err := os.WriteFile(winutils.GetHostPath("/host/etc/cni/net.d/calico-kubeconfig"), []byte(data), 0600); err != nil {
 		logrus.Fatal(err)
 	}
 }
 
 func setSuidBit(file string) error {
+	if runtime.GOOS == "windows" {
+		// chmod doesn't work on windows
```

Comment:
Should we include a log of some sort here so that we have an idea why it quit early?

Reply:
Will do, thanks!